### DISCUSSION TITLE
Supported Azure image for tpotce?

### DISCUSSION THREAD

**Discussion by sdfdsfsdfdsfsdf** [telekom-security/tpotce#1493] [Q&A]
I keep getting the same error from numerous VMs I made through Azure, that the <selected image> is not supported. I was wondering what actually _is_ the supported image to use tpotce with Microsoft Azure. 

The images below show the last one I made before I gave up. 

![Screenshot 2024-03-18 004558](https://github.com/telekom-security/tpotce/assets/161926573/4f6e6e4c-c829-47c0-a659-8e3f1f4a1b67) _Using MobaXTerm as an SSH client._

![Screenshot 2024-03-18 005019](https://github.com/telekom-security/tpotce/assets/161926573/516e4181-2b47-4ff7-bd1f-5c20dbd50027)
_Taken from Azure, this is specs for Debian 10.13 which was the last image I tested._

All the help is appreciated :)

**Comment by t3chn0m4g3** [comment]
Stay tuned #1487
